src/main/java/net/minecraftforge/fml/common/registry/VillagerRegistry.java

```diff
@@ -160,7 +162,6 @@ public void registerVillageCreationHandler(IVillageCreationHandler handler)
      * @param defaultSkin
      */
     @SideOnly(Side.CLIENT)
-    @Deprecated // Doesn't work at all.
```

Comment:
should I add a replacement method which takes the VillagerProfession and the texture instead?